[2024-01-31T14:47:23.971+0500] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.hello_task manual__2024-01-31T09:47:22.208098+00:00 [queued]>
[2024-01-31T14:47:23.982+0500] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.hello_task manual__2024-01-31T09:47:22.208098+00:00 [queued]>
[2024-01-31T14:47:23.983+0500] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-31T14:47:23.995+0500] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): hello_task> on 2024-01-31 09:47:22.208098+00:00
[2024-01-31T14:47:24.004+0500] {standard_task_runner.py:60} INFO - Started process 97306 to run task
[2024-01-31T14:47:24.010+0500] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'hello_task', 'manual__2024-01-31T09:47:22.208098+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/var/folders/zx/n46yb4dj7k1_ltz_w7pn0tmh0000gn/T/tmpd06hz8bd']
[2024-01-31T14:47:24.013+0500] {standard_task_runner.py:88} INFO - Job 401: Subtask hello_task
[2024-01-31T14:47:24.104+0500] {task_command.py:423} INFO - Running <TaskInstance: my_dag.hello_task manual__2024-01-31T09:47:22.208098+00:00 [running]> on host syeds-macbook-pro.local
[2024-01-31T14:47:24.329+0500] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T09:47:22.208098+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T09:47:22.208098+00:00'
[2024-01-31T14:47:24.330+0500] {logging_mixin.py:188} INFO - Hello, World!
[2024-01-31T14:47:24.330+0500] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T14:47:24.341+0500] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=hello_task, execution_date=20240131T094722, start_date=20240131T094723, end_date=20240131T094724
[2024-01-31T14:47:24.392+0500] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T14:47:24.416+0500] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
